### ISSUE TITLE
koderimakl

### ISSUE THREAD

**Issue by johnbent** [tekinged/missing#414]

**Comment by Gnalu** [comment]
Same as ‘kedrimakl’ or ‘kederimakl’. 
A Ulang an Mla kederimakl er a chelsel a blai  ediak lorrous a ilunel. Ulang has gotten in the house and not serving drinks.

**Comment by s-o-u-a-n-g** [comment]
Right. Variation in pronunciation. I hope there's a disclaimer available that the dictionary is not a repository of all variations in pronunciation and absolute definitions since language is flexible, contextual, and evolving. 

I sometimes get messages asking if a word is spelled this way or that way. I usually say "YES". It is initially confusing but then I let them know they should spell it how it is said in their family or village. Then I share how I think it's spelled. And then, lastly, I tell them how it's spelled in the dictionary.

**Comment by johnbent** [comment]
LOL @ "I usually say "YES"".

Have I complained yet about the number of people who complain about missing
English words? :)

**Comment by s-o-u-a-n-g** [comment]
That's a bit over my head. People are complaining about missing English words? Like wanting to add non-English words to the English dictionary?

Oh. Thinking further I think you mean English words in the English part of the Palauan-English dictionary? 

Often, too often, when people ask me "what is the English word for mengebou?" Or, "what is the Palauan word for identity theft?" I counter with "Maybe we can ask *IS* there such a word?" because asking "what" is the word already establishes that there IS one where none may exist. 

Similarly I have been asked "What is the national dish of Palau?" and I responded "I know your question is a simple and straightforward one, however, before I answer, maybe we can think about how we ask questions. For example, we can ask *IS* there a national dish of Palau?"

They were taken aback but I know it made an impact. Later I said "Palauans usually eat odoim and ongraol and sometimes kliou." And the puzzled look was responded to with "Palauans usually eat a protein based dish and a carbohydrate based fish and sometimes we eat dessert."

I'd love to know what words you've been bothered with to add! If it's appropriate to share, of course.

**Comment by johnbent** [comment]
Hey @s-o-u-a-n-g , I think it's totally appropriate to share. But I don't know how to export from gmail so I'm just attaching screen shots:

<img width="199" alt="image" src="https://github.com/tekinged/missing/assets/2047294/0961eefe-f399-4de0-843b-04af8328cc92">
<img width="318" alt="image" src="https://github.com/tekinged/missing/assets/2047294/c635592a-b0df-4bc6-a6af-8fe97695db05">

I'm happy so many people are using the dictionary! There does seem to be some confusion about how language works. :)
